grant-project/zcash-grant-system
# Sentry fixes

Closes #313.

### What This Does

* Upgrades all sentry versions to fix some issues we've had (all the python errors, circular JSON)
* Replace python logging `print` statements with `app.logger` for formatted logging
    * Debug statements aren't sent with Sentry errors, so they're usually for redundant or low-interest information. Every other log is sent as "breadcrumbs" to help deduce issues.
* Add a app.errorhandler for 404's to prevent those from going to Sentry

### Steps to Test

Browse around and make sure everything's kosher. If you want to get spicy, throw an error and make sure it still logs. Check out the breadcrumbs too, very cool stuff.
